src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -103,6 +104,22 @@ public String getUrlForWiki(String wikiName, boolean addWWW) {
     }
   }
 
+  public  String getWikiGlobalURL(){
+    EnvType env = Configuration.getEnvType(this.env);
+
+    switch (env) {
+      case DEV: {
+        String devBoxOwner = this.env.split("-")[1];
+        return String.format("http://%s.www.%s", devBoxOwner, env.getWikiaDomain());
+      }
+      case SANDBOX: {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1199)